### PR TITLE
index overview

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
-TODO
+## Native indexes
+
+Native indexes allow querying data based on a given property. There are two kinds of native indexes: tag index and edge type index. Native indexes must be updated manually. You can use the `REBUILD INDEX` statement to update native indexes. Native indexes support indexing multiple properties on a tag or an edge type (composite indexes), but do not support indexing across multiple tags or edge types.
+
+You can do partial match search by using composite indexes. Only use composite indexes for partial match searches when the declared fields in the composite index are used from left to right. For more information, see [LOOKUP FAQ](../7.general-query-statements/5.lookup.md#FAQ).
+
+String operators like `CONTAINS`, `STARTS WITH` are not allowed in native index searching. Use full-text indexes to search.
+
+### Operations against native indexes
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:13]: Operations on native indexes

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+- [Rebuild index](4.rebuild-native-index.md)
+- [Show index status](5.show-native-index-status.md)
+- [Drop index](6.drop-native-index.md)
+- [Query index](../7.general-query-statements/5.lookup.md)
+
+## Full-text indexes
+
+Full-text indexes are used to index the string properties for vertices or edges. A full-text index allows you to write queries to retrieve data within the contents of indexed string properties. You can use full-text index to do prefix, wildcard, regexp, and fuzzy search on string property. Full-text indexes do not support indexing multiple properties a tag or an edge type.
+
+### Operations against full-text indexes
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:29]: on

**Review by whitewum — APPROVED** [review]
minor and oK

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+- [Show index](2.show-native-indexes.md)
+- [Describe index](3.describe-native-index.md)
+- [Rebuild index](4.rebuild-native-index.md)
+- [Show index status](5.show-native-index-status.md)
+- [Drop index](6.drop-native-index.md)
+- [Query index](../7.general-query-statements/5.lookup.md)
+
+## Full-text indexes
+
+Full-text indexes are used to index the string properties for vertices or edges. A full-text index allows you to write queries to retrieve data within the contents of indexed string properties. You can use full-text index to do prefix, wildcard, regexp, and fuzzy search on string property. Full-text indexes do not support indexing multiple properties a tag or an edge type.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:27]: You can use full-text indexes

**Review by bright-starry-sky — APPROVED** [review]

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
 
-Full-text indexes are used to index the string properties for vertices or edges. A full-text index allows you to write queries to retrieve data within the contents of indexed string properties.
+Indexes are built to fast process graph queries. Nebula Graph supports two kinds of indexes: native indexes and full-text indexes. This topic introduces the index types and helps choose the right index.
 
-TODO
+## Native indexes
+
+Native indexes allow querying data based on a given property. There are two kinds of native indexes: tag index and edge type index. Native indexes must be updated manually. You can use the `REBUILD INDEX` statement to update native indexes. Native indexes support indexing multiple properties on a tag or an edge type (composite indexes), but do not support indexing across multiple tags or edge types.
+
+You can do partial match search by using composite indexes. Only use composite indexes for partial match searches when the declared fields in the composite index are used from left to right. For more information, see [LOOKUP FAQ](../7.general-query-statements/5.lookup.md#FAQ).
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:9]: Use composite indexes only for?

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+
+String operators like `CONTAINS`, `STARTS WITH` are not allowed in native index searching. Use full-text indexes to do fuzzy search.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:11]: `CONTAINS` and `STARTS WITH` ?

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 6]
```diff
@@ -1,5 +1,39 @@
...
+
+### Operations on native indexes
+
+You can do the following operations against native indexes:
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:15]: against --> on

File: docs-2.0/3.ngql-guide/14.index-statements/README.md [hunk 7]
```diff
@@ -1,5 +1,39 @@
...
+- [Show index](2.show-native-indexes.md)
+- [Describe index](3.describe-native-index.md)
+- [Rebuild index](4.rebuild-native-index.md)
+- [Show index status](5.show-native-index-status.md)
+- [Drop index](6.drop-native-index.md)
+- [Query index](../7.general-query-statements/5.lookup.md)
+
+## Full-text indexes
+
+Full-text indexes are used to do prefix, wildcard, regexp, and fuzzy search on a string property. Full-text indexes allow indexing just one property. Only strings within a specified (no longer than 256 bytes) length are indexed. Full-text indexes do not support logical operations such as `AND`, `OR` and `NOT`. To do complete text match, use native indexes.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:27]: and fuzzy  --> or fuzzy?

Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/README.md:27]: "a specified (no longer than 256 bytes) length" --> "a specified length (no longer than 256 bytes)"

File: docs-2.0/3.ngql-guide/7.general-query-statements/5.lookup.md [hunk 8]
```diff
@@ -32,7 +32,7 @@ LOOKUP ON {<vertex_tag> | <edge_type>} [WHERE <expression> [AND <expression> ...
 
 The `WHERE` clause in a `LOOKUP` statement does not support the following operations:
 
-- `$-` and `$^`
+- `$-` and `$^`.
 - In relational expressions, expressions with field names on both sides of the operator are not supported, such as `tagName.column1> tagName.column2`.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/5.lookup.md:36]: should we replace "column" with "property"?

**Review by bright-starry-sky — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**